news/_posts/2021-07-06-responsivebo.md

```diff
@@ -0,0 +1,132 @@
+--- 
+layout: post
+title:  "Towards a more mobile-friendly back office"
+subtitle: "Quick wins of the responsive back office"
+date:   2021-07-06 09:00:00
+authors: [ScottCosio]
+icon: icon-mobile
+tags:
+ - 1.7.8
+ - version
+ - minor
+ - product
+ - ux
+---
+
+Until now, PrestaShop has always relied on a desktop-first responsive design strategy. Indeed, there are few uses cases on mobile. But you can't prevent users from checking their store's back office on their mobile!
+
+## Why work on responsive design?
+
+Today, there are some issues that affect the user experience on mobile device:
+
+- Difficulties in accessing important store informations
+- Hidden contents because of components overlaying others
+- Text readability issues
+- Bugs and lack of consistency on display
+
+All of these issues cause a bad reputation for the PrestaShop open source project.
+
+## First steps: quick wins 💪🏻
+
+Valentin (front-end developer), and I, Scott (product designer), didn't have much time to work on this project. So, to avoid breaking changes, we updated the back office mobile version thanks to "quick wins".
+
+> A quick win is a small change, that is quick and easy to implement but that has a big and visible impact for the user.
+
+### Process
+
+The Core team first explored PrestaShop issues about responsive design on GitHub and centralized them by creating a [single Epic.](https://github.com/PrestaShop/PrestaShop/issues/22358)
+
+In parallel, we explored all back office pages on mobile version and discussed a lot about software responsiveness. For each problem, a specific issue was created and added to the Epic. 
+
+The Epic was organized in 3 different categories:
+
+- Remove components - components that aren't important or used in mobile version
+- Improvements - optimizing navigation, optimizing space, etc
+- Bugs - display adjustments, broken pages, etc
+
+### What has changed?
+
+**Removing components**
+
+Some components are available on mobile version but aren't necessarily useful and become easily annoying for the user experience.
+
+- The "Recommended Modules and Services" button has been removed because of the PrestaShop open source projects' split. It's not available anymore.
+- The "Import" and "Export" buttons have been removed for tables because importing or exporting .csv files is better on larger screens and wasn't something usually handled on mobile version.
+- Breadcrumbs have been removed because it's not really necessary since there is barely more than 2 depth levels.
+- In the header, "View my shop" text has been removed to shorten the button and save space in the header/top bar. The Core team decided to only keep the icon.
```

Comment:
```suggestion
- In the header, "View my shop" text has been removed to shorten the button and save space in the header/top bar. Only the icon was kept.
```
To avoid using "The Core team" too much. :-)